test/spec/util/util.js

```diff
@@ -230,6 +230,13 @@ describe('iD.util', function() {
         it('returns the name if tagged with a name', function() {
             expect(iD.utilDisplayName({tags: {name: 'East Coast Greenway'}})).to.eql('East Coast Greenway');
         });
+        it('returns just the name for non-routes', function() {
+            expect(iD.utilDisplayName({tags: { name: 'Abyssinian Room', ref: '260-115' }})).to.eql('Abyssinian Room');
+        });
+        it('returns the name and the ref for routes', function() {
+            expect(iD.utilDisplayName({tags: { name: 'Lynfield to Midtown', ref: '25L', route: 'bus' }})).to.eql('25L: Lynfield to Midtown');
```

Comment:
We no longer want to encourage the use of `name` for route descriptions, now that #8276 has landed. Here’s an example of a name that would remain after all the route descriptions are removed from `name` tags:

```suggestion
            expect(iD.utilDisplayName({tags: { name: 'Lynfield Express', ref: '25L', route: 'bus' }})).to.eql('25L: Lynfield Express');
```